### DISCUSSION TITLE
Removing 'admin' subdomain from login route

### DISCUSSION THREAD

**Discussion by iliabaan** [area17/twill#1146] [Q&A] [answered]
Hello there! 
I'm trying to remove 'admin' subdomain from login route like 'domain.test/login' instead of 'domain.test/admin/login'. 
Any ideas?

**Comment by pboivin** [comment, marked as answer]
Hi @iliabaan,

The most simple way to handle this would be with a redirect:

```
// add to file: routes/web.php

Route::redirect('/login', '/admin/login');
```

This will not hide the `/admin` prefix but it'll give you the option of sharing a "cleaner" URL, like `myapp.com/login`.


You could also add `/login` as an alias to `/admin/login`, without needing to override anything from Twill:

```
// add to file: routes/web.php

use A17\Twill\Http\Controllers\Admin\LoginController;

Route::get('login', [LoginController::class, 'showLoginForm']);
```

**Reply by iliabaan** [reply]
Thank you!
